crt_portal/cts_forms/tests/test_views.py

```diff
@@ -876,6 +876,92 @@ def test_complaint_id_filter(self):
         self.assertTrue('2 records' in str(response.content))
 
 
+class CRT_Disposition_Tests(TestCase):
```

Comment:
nit: I know this is how the rest of the file is named, but can we use regular class naming convention? I'd like to change the others at some point, too.

```suggestion
class CrtDispositionTests(TestCase):
```